host_vars/OCT4-SW-TORS/interfaces.yaml

```diff
@@ -0,0 +1,330 @@
+interfaces:
+  1/1:
+    description: "OCT4-17"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/2:
+    description: "OCT4-16"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/3:
+    description: "OCT4-15"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/4:
+    description: "OCT4-14"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/5:
+    description: "OCT4-13"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/6:
+    description: "OCT4-12"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/7:
+    description: "OCT4-11"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/8:
+    description: "OCT4-10"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/9:
+    description: "OCT4-9"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/10:
+    description: "OCT4-8"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/11:
+    description: "OCT4-7"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/12:
+    description: "OCT4-6"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/13:
+    description: "OCT4-5"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/14:
+    description: "OCT4-4"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
+  1/15:
+    description: "OCT4-3"
+    admin: "up"
+    mtu: 9216
+    managed-l2: true
```

Comment:
oct4-{3,4,5} are not managed-l2 as they are ESI overcloud controllers themselves. oct4-0 is the undercloud so that's not also not managed-l2.

some old comment where I configured the switchports for these machines

https://github.com/CCI-MOC/esi/issues/176#issuecomment-1022505663

Reply:
Okay, makes sense. I am leaving OCT4-2 and -1 as managed-l2. Is that correct? Also, are all of the secondary ports unmanaged as well? I think just 0-5 have dual links.

Reply:
yeah, the secondary ports are also unmanaged as well.

Reply:
sorry, I missed the "all" of "are all of the secondary ports unmanaged as well?"

Secondary ports of the overcloud controllers and the undercloud are unmanaged. 

Any other node, if it's registered in ESI and has switchports on an ESI managed switch should be assumed to be managed by ESI. So I guess, whatever dual link OCT4 node that's left will have it's ports managed.